definition.go

```diff
@@ -181,33 +188,39 @@ func GetNamed(ttype Type) Named {
 	return unmodifiedType
 }
 
-/**
- * Scalar Type Definition
- *
- * The leaf values of any request and input values to arguments are
- * Scalars (or Enums) and are defined with a name and a series of functions
- * used to parse input from ast or variables and to ensure validity.
- *
- * Example:
- *
- *     var OddType = new Scalar({
- *       name: 'Odd',
- *       serialize(value) {
- *         return value % 2 === 1 ? value : null;
- *       }
- *     });
- *
- */
+// Scalar Type Definition
+//
+// The leaf values of any request and input values to arguments are
+// Scalars (or Enums) and are defined with a name and a series of functions
+// used to parse input from ast or variables and to ensure validity.
+//
+// Example:
+//
+//    var OddType = new Scalar({
+//      name: 'Odd',
+//      serialize(value) {
+//        return value % 2 === 1 ? value : null;
+//      }
+//    });
```

Comment:
@jvatic That's a great idea 👍

Should we address this is in a separate PR? It would be great to get help with refining the documentation (currently golint complains that there are too many exported variables/methods/structs etc that are missing comments)